test/functional/cmdLineTests/criu/criu_nonPortable.xml

```diff
@@ -327,4 +327,21 @@
     <!-- In the past, the failure below was caused by an issue where CRIU can't be found on the PATH. -->
     <output type="failure" caseSensitive="yes" regex="no">Could not dump the JVM processes, err=-70</output>
   </test>
+  <test id="Trace Options Test">
+    <command>bash $SCRIPPATH$ $TEST_RESROOT$ $JAVA_COMMAND$ "$JVM_OPTIONS$" $MAINCLASS_OPTIONSFILE_TEST$ TraceOptionsTest 1</command>
```

Comment:
we should add a couple tests that starts with traceoptions and adds new ones (system and java method tracing)

Reply:
Added two tests w/ `-Xtrace:print={j9jcl.219}` & `-Xtrace:print=mt,methods=org/eclipse/openj9/criu/CRIUSupport.registerRestoreOptionsFile()` before the checkpoint. Another `criu_nonPortable_Xtrace.xml` is introduced to include these two options within `playlist.xml` just like https://github.com/eclipse-openj9/openj9/blob/9464e1bc8b8e5fe1cca073809f4a8898c9eb7a88/test/functional/cmdLineTests/criu/playlist.xml#L98